src/jit/importer.cpp

```diff
@@ -7715,7 +7715,7 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
                         unsigned   calliSlot  = lvaGrabTemp(true DEBUGARG("calli"));
                         LclVarDsc* varDsc     = &lvaTable[calliSlot];
                         varDsc->lvVerTypeInfo = tiRetVal;
-                        impAssignTempGen(calliSlot, call, clsHnd, (unsigned)CHECK_SPILL_NONE);
+                        impAssignTempGen(calliSlot, call, tiRetVal.GetClassHandle(), (unsigned)CHECK_SPILL_NONE);
```

Comment:
I think it gets stomped over by `clsHnd = pResolvedToken->hClass;` later.

Plus I'm seeing this on the GVM codepath anyway (so not CEE_CALLI, I think).